sbc_compassion/views/import_letters_history_view.xml

```diff
@@ -8,29 +8,27 @@
         <field name="arch" type="xml">
             <form string="Imports">
                 <header>
-                    <field name="state" widget="statusbar" statusbar_visible="draft,open,done"/>
+                    <field name="state" widget="statusbar" statusbar_visible="draft,pending,open,done"/>
                     <field name="import_completed" invisible="1"/>
                     <button name="button_save" string="Import Letters" states="ready" type="object" class="oe_highlight"/>
-                    <button name="button_import" string="Load Files" type="object" states="draft"/>
+                    <button name="button_import" string="Load Files" type="object"/>
```

Comment:
Why removing the state filter? I think the button makes no sense when the import is not in draft state.